metrics/zz_generated.deepcopy.go

```diff
@@ -0,0 +1,37 @@
+// +build !ignore_autogenerated
+
+/*
+Copyright 2019 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Code generated by deepcopy-gen. DO NOT EDIT.
```

Comment:
Did you manually create this file?

Reply:
no, moved it from `serving/pkg/metrics/`

Reply:
You shouldn't even have to manually move it.  You probably need to update `./hack/update-codegen.sh` to run deepcopy-gen on this directory.

Reply:
sounds good thanks 👍